### PR TITLE
Main header

### PR DESCRIPTION
**Pull request by mateoclarke**
screen shot 2017-08-24 at 1 45 10 am

### CODE REVIEW

**Comment by mateoclarke** [comment]
@briaguya, This is ready for review. I merged in #24, but this PR is still showing files that are already in master. 🤷‍♂️ 

My issues doing PR's from my fork vs against the main repo should be ok going forward. Sorry I suck.

This commit is the only thing new after PR #24
https://github.com/cityofaustin/ctxfloods/pull/25/commits/c6df84aa4984a0c178fbea6ed22200b5d1fa3c95

File: frontend/src/Button.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.Button {
```
Comment (briaguya) [inline frontend/src/Button.css]: I'm assuming this is LinkButton now

File: frontend/src/Button.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+import './Button.css';
+
+class Button extends Component {
```
Comment (briaguya) [inline frontend/src/Button.js]: LinkButton too?

File: frontend/src/Button.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.Button {
```
Comment (briaguya) [inline frontend/src/Button.scss]: Link! HE COME TO BUTTON!

File: frontend/src/Header/Header.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
+.flexcontainer {
```
Comment (briaguya) [inline frontend/src/Header/Header.css]: Overall question, if we're doing the scss stuff does it make sense to check in the css files?

Maybe we should ignore them to make sure people use the generated files?

File: frontend/src/Header/UserControls.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+import auth from '../services/gqlAuth'
+
+import cogSvg from '../images/cog.svg';
+import './UserControls.css'
+
+const UserControls = withRouter(({ history }) => (
+  auth.isAuthenticated() ? (
+    <div className="UserControls">
+      <img src={cogSvg} alt="Link User Settings" className="Header__settings-icon" />
+      <Link to="#">Matthew Porcher</Link>
```
Comment (briaguya) [inline frontend/src/Header/UserControls.js]: do we have a story for making this link work in the backlog yet? If not we should make one.

File: frontend/src/Header/Header.css (hunk 4 above)
Comment (mateoclarke) [reply frontend/src/Header/Header.css]: yeah, I think that's a good call. Crossed my mind too.

File: frontend/src/Header/UserControls.js (hunk 5 above)
Comment (mateoclarke) [reply frontend/src/Header/UserControls.js]: I'll add it to the backlog.

**Pull request merged**